pages/1.9/release-notes/1.9.0/index.md

```diff
@@ -6,6 +6,8 @@ menuWeight: 60
 excerpt:
 ---
 
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.9.0/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
```

Comment:
For some reason, this versioned link doesn't work. The old release notes point to this instead (which is the tagged commit in dcos/dcos): https://downloads.dcos.io/dcos/stable/commit/0ce03387884523f02624d3fb56c7fbe2e06e181b/dcos_generate_config.sh